### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by ziyi**

### CODE REVIEW

**Comment by khatchad** [comment]
Did I give you a grade for this in Blackboard?

**Comment by ziyi** [comment]
No I don't see a grade item for HW1

**Comment by khatchad** [comment]
What's your name?

**Comment by ziyi** [comment]
Ziying Guo

**Comment by khatchad** [comment]
I don't have a submission for you on Blackboard. Did you submit your username there?

File: src/hw1/StringManipulator.java [hunk 1]
```diff
@@ -20,5 +21,16 @@ public static void main(String[] args) {
          The name of the city in all lowercase letters
          The first character in the name of the city
          */
+            System.out.println("please enter the name of your favorite city:");
```
Comment (khatchad) [inline src/hw1/StringManipulator.java:24]: Please format your code prior to submission.

Comment (ziyi) [inline src/hw1/StringManipulator.java:24]: Do I have to redo this?

Comment (khatchad) [inline src/hw1/StringManipulator.java:24]: No but please do it next time.

**Pull request closed without merging**